recipes/scikit-geometry/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "scikit-geometry" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/scikit-geometry/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 89f7d3a6795d657cdf75b0c3d20d8929765e4ea8eae6da6a61edb538c9664603
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
```

Seems to require some compilers
```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```